api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImpl.java

```diff
@@ -41,7 +43,7 @@ public Patient getPatientById(Integer id) {
 	}
 	
 	@Override
-	public Patient getPatientByUuid(String uuid) {
+	public Patient get(String uuid) {
```

Comment:
Technically, you should be able to remove this entire method, as its implemented in `BaseFhirDao` now.